### DISCUSSION TITLE
DockerImage usage

### DISCUSSION THREAD

**Discussion by jordy-dam-inqdo** [ServerlessLife/lambda-live-debugger#79] [Q&A]
Hi there,

My question is how can I use this if my lambda's are running as Docker images? Do I need to include the extension in the /opt/extensions? Do you have a sample on how to do this?

Thanks!
J.

**Comment by ServerlessLife** [comment]
Hi @jordy-dam-inqdo,

Thank you for posting the questions.

I am sorry. Debugging Lambdas that are deployed as containers is not supported.

I am curious what is the reason that you are using containers.

Marko

**Comment by jordy-dam-inqdo** [comment]
Some of our clients are using containers, and with a team of developers requested the possibility of live debugging. I came across your repo and seemed interesting, unfortunately we have containers so we either have to create the extension as a separate docker image and add it to the lambda's dockerfile or include it directly. I am not sure we will be able to talk them out of using containers

**Comment by ServerlessLife** [comment]
Hi @jordy-dam-inqdo,

Thanks for the additional information.

If more users were interested in containers, I might consider supporting them. But for now, it looks more like a niche use case.

**Comment by jordy-dam-inqdo** [comment]
Understandable, thanks for the reply! And keep up the nice work with the debugger!
